bms/bms_app/static/bms_app/css/registerStyle.css

```diff
@@ -2,53 +2,38 @@ body{
     background-image:url('/static/bms_app/images/templates.jpg');
     background-attachment:fixed;
     background-size:cover;
-    overflow-y: hidden;
 }
+
 .panel-title{
     text-align: center;
     font-weight: bold;
     font-size: 30px;
 }
+
 #loginbox{
```

Comment:
why do we need multiple style sheet for same layout.? in my earlier review I already recommended not to use multiple style sheets for same layout.